api/src/main/java/com/flipkart/poseidon/validator/BlocksValidator.java

```diff
@@ -33,55 +33,58 @@
     private static final Logger logger = LoggerFactory.getLogger(BlocksValidator.class);
 
     public static void main(String[] args) {
-        try {
-            final String customValidatorClass = System.getProperty("poseidon.validator.block.custom");
-            CustomBlocksValidator customValidator = null;
-            if (StringUtils.isNotEmpty(customValidatorClass)) {
-                try {
-                    final Class<?> aClass = Class.forName(customValidatorClass);
-                    final Object constructedInstance = aClass.newInstance();
-                    if (constructedInstance instanceof CustomBlocksValidator) {
-                        customValidator = (CustomBlocksValidator) constructedInstance;
-                    } else {
-                        throw new IllegalArgumentException("Wrong class supplied");
+        boolean skipBlocksValidator = Boolean.valueOf(System.getProperty("skipBlocksValidator"));
```

Comment:
Same as above for APIValidator, this would be invoked by client app during the compile phase using mvn exec plugin in datasources module.